src/chef-server-ctl/plugins/notice.rb

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright:: Copyright (c) Chef Software, Inc.
+#
+# All Rights Reserved
+#
+
+add_command_under_category "notice", "general", "Display important notice and license information.", 2 do
+  notice_message = <<~NOTICE
+    SHAHID ---> You are good
+    Please accept all the license here written
+    Please consider this notice
```

Comment:
This is a public repo, please be careful in putting random text.
Replace it with actual NOTICE.txt content.

Also, keep NOTICE.txt in the root dir, if possible. (for standardisation across repos)